integration/Dockerfile

```diff
@@ -1,23 +1,30 @@
 # This Dockerfile builds a base image for the CloudBuild integration testing.
 # It just extends a golang/linux base image with a few extra tools which the
 # tests need.
-FROM golang:1.15 as testbase
+FROM ubuntu as testbase
 
 WORKDIR /testbase
 
 ARG GOFLAGS=""
 ENV GOFLAGS=$GOFLAGS
 ENV GO111MODULE=on
 
-RUN apt-get update && apt-get -y install curl unzip wget default-mysql-client docker-compose lsof xxd
+RUN apt-get update && apt-get -y install curl default-mysql-client docker-compose lsof unzip wget xxd
```

Comment:
I wonder what we're using the mysql-client for?

Reply:
Ah, it used to be for the resetDB stuff, but I think that's gone now.